applications/GeoMechanicsApplication/custom_utilities/element_utilities.hpp

```diff
@@ -604,6 +606,9 @@ class GeoElementUtilities
         return nodal_hydraulic_heads;
     }
 
+    static std::vector<Vector> CalculateShapeFunctionValues(const Geo::IntegrationPointVectorType& rIntegrationPoints,
```

Comment:
We may consider to make this function name slightly more specific, e.g. `EvaluateShapeFunctionsAtIntegrationPoints`.